app/config/tstvhq/adventures.json

```diff
@@ -26,60 +26,60 @@
                 },
                 {
                     "id": "catacombs",
-                    "name": "An Unpredictable Stair",
-                    "text": "You reach out and right the painting.  An unsettling ratcheting sound echoes in the wall and suddenly the stair you are standing on trundles aside. You tumble into a slide that carries you through cold, dark air, bumping and thumping as you round awkward turns, until it dumps you into a subterranean chamber. A fortune telling machine sits ominously in the center of the rock-hewn floor, and on the machine is a button. There is also a door to your right.",
+                    "name": "Through the looking glass",
+                    "text": "You appear to be the only one who sees that you have inadvertantly opened a secret door. You decend down the spiral staircase within. The door closes behind you. Decending, you find yourself in an ante-chamber with a single door.",
                     "buttons": [
                         {
-                            "text": "Push the button",
+                            "text": "Open the door",
                             "nextRoom": ["fortune"]
                         },
                         {
-                            "text": "Go through the door",
+                            "text": "Look for a way out",
                             "nextRoom": ["library"]
                         }
                     ]
                 },
                 {
                     "id": "fortune",
-                    "name": "Fortune",
-                    "text": "The machine whirs and rumbles to life. Eyes flashing, the fortune teller says in a lilting Indian accent, ”My name is Madame la Vache. I sense the presence of someone seeking knowledge.  I will now divine your future!“ A card pokes out from a slot below the machine and her eyes grow dim once again. She lurches back into an awkward pose and is silent.",
+                    "name": "Open the door",
+                    "text": "The door will not open. It is locked. You can hear it though.",
                     "buttons": [
                         {
-                            "text": "Pick up the card",
+                            "text": "stop and listen",
                             "nextRoom": ["card"]
                         },
                         {
-                          "text": "Go through the door",
+                          "text": "Look for a way out",
                           "nextRoom": ["library"]
                       }
                     ]
                 },
                 {
                     "id": "card",
-                    "name": "The Card",
-                    "text": "For possibly the first time, divination has correctly told you something true.  The card reads, ”You are going to die someday.“",
+                    "name": "listen",
+                    "text": "You listen.“",
```

Comment:
not technically a syntax error since directional quotes don't count, but it's damn confusing

Reply:
fixed